components/org.wso2.carbon.identity.oauth.dcr/pom.xml

```diff
@@ -165,6 +168,7 @@
                             org.wso2.carbon.identity.application.common.model;version="${carbon.identity.framework.imp.pkg.version.range}",
                             org.wso2.carbon.identity.application.mgt.*;version="${carbon.identity.framework.imp.pkg.version.range}",
                             org.wso2.carbon.identity.application.authentication.framework.*;version="${carbon.identity.framework.imp.pkg.version.range}",
+                            org.wso2.carbon.identity.organization.management.service.*; version="${carbon.identity.organization.management.core.version.range}",
```

Comment:
I think it is the best practice to use specific imports rather than * imports.

Reply:
yes specifically mention only required packages

Reply:
updated